src/aggregate.cc

```diff
@@ -34,21 +34,27 @@ Napi::Value AggregatePublicKeys(const Napi::CallbackInfo &info) {
                         env, "BLST_ERROR: PublicKeyArg"s + *err_msg)
                         .ThrowAsJavaScriptException();
                     has_error = true;
-                } else if (blst_ts::is_zero_bytes(
-                               typed_array.Data(),
-                               0,
-                               typed_array.ByteLength())) {
-                    Napi::TypeError::New(
-                        env, "BLST_ERROR: PublicKeyArg must not be zero key")
-                        .ThrowAsJavaScriptException();
-                    has_error = true;
                 } else {
-                    aggregate.add(
-                        blst::P1{typed_array.Data(), typed_array.ByteLength()});
+                    blst::P1 point{
+                        typed_array.Data(), typed_array.ByteLength()};
+                    if (point.is_inf()) {
```

Comment:
This is one of the conditions that triggered the refactor of the code.  I was checking for all 0x00 bytes and there is a 3-bit header for informing how to interpret byte strings during deserialization.  In the JS glue code we were checking for zero keys so the check I had at the native layer never got tested and was built incorrectly because the first byte is never 0x00 because of the header.

Added a comment.